whylog/client/searchers.py

```diff
@@ -0,0 +1,56 @@
+from abc import ABCMeta, abstractmethod
+from os import SEEK_SET
+
+from whylog.client.const import BufsizeConsts
+
+
+class AbstractSearcher(object):
+
+    __metaclass__ = ABCMeta
```

Comment:
This doesn't work on Python3. See `whylog/constraints/__init__.py` on how to do it using six